cmd/jwt.go

```diff
@@ -43,6 +43,7 @@ var errInvalidSecretKeyLength = errors.New("Invalid secret key, secret key shoul
 
 var errInvalidAccessKeyID = errors.New("The access key ID you provided does not exist in our records")
 var errAuthentication = errors.New("Authentication failed, check your access credentials")
+var errNoAuthToken = errors.New("JWT token missing")
```

Comment:
we don't call toJSONError(errNoAuthToken) anywhere hence it's not needed. This error value is mainly needed to differentiate between errNoAuthToken and errAuthentication so that we reply with "auth error" when call is made with incorrect token on a, for example, public bucket instead of serving the request.